src/components/Debate/DebateView.tsx

```diff
@@ -52,6 +63,24 @@ const DebateView = ({ claim }) => {
                         title={claim?.title}
                         personalities={claim?.personalities}
                     />
+                    {userRole === Roles.Regular ? null : (
+                        <AletheiaButton
+                            type={ButtonType.blue}
+                            style={{
+                                width: "10%",
+                                margin: 10,
+                                paddingLeft: 15,
+                            }}
+                            href={href}
+                        >
+                            <span>{t("debates:openEditDebateMode")}</span>
+                            <EditOutlined
+                                style={{
+                                    color: "#ffffff",
```

Comment:
chore (non-blocking): utilize our colors file library `colors.white`, and import it from "/styles/colors.ts